package.json

```
@@ -36,66 +36,96 @@
       "name": "Michael Prentice",
       "email": "michael@xlts.dev",
       "url": "https://github.com/splaktar"
+    },
+    {
+      "name": "Jaroslaw Zolnowski",
+      "email": "jarek@xlts.dev",
+      "url": "https://github.com/jzolnowski"
     }
   ],
-  "main": "index.js",
   "scripts": {
-    "start": "webpack-dev-server --open --mode development",
-    "test": "karma start karma.config.js",
-    "build": "webpack --mode production",
-    "prettier": "prettier --write \"**/*.{js,json,css,scss,less,md,ts,html}\"",
+    "pre-commit": "lint-staged",
+    "build:templates": "gulp templates",
+    "start:server": "webpack-dev-server --open --mode development --config config/webpack.dev.mjs",
+    "watch:templates": "gulp watch",
+    "start": "npm run build:templates && run-p start:server watch:templates",
     "lint": "eslint .",
-    "update": "npx npm-check -u",
-    "prepare": "husky install"
+    "test:unit": "karma start karma.config.js",
+    "test:unit:watch": "npm run build:templates && run-p test:unit watch:templates",
+    "test:unit:single": "run-s build:templates \"test:unit -- --single-run\"",
+    "build": "npm run build:templates && webpack --mode production --config config/webpack.prod.mjs",
+    "start:prod-local": "http-server dist -p 4200 -c-1 -b",
+    "prettier": "prettier --write \"**/*.{js,json,css,scss,less,md,ts,html}\"",
+    "test:e2e": "playwright test",
+    "test:e2e:debug": "playwright test --project chromium --headed",
+    "test:e2e:report": "npx playwright show-report",
+    "test": "run-s build:templates lint test:unit:single test:e2e",
```

Unless I am missing something, we shouldn't need explicitly building the templates here:

```suggestion
    "test": "run-s lint test:unit:single test:e2e",
```